lib/arborist/build-ideal-tree.js

```diff
@@ -234,9 +237,19 @@ module.exports = cls => class IdealTreeBuilder extends Tracker(Virtual(Actual(cl
       optional: false,
       global: this[_global],
       legacyPeerDeps: this.legacyPeerDeps,
+      hasWorkspaces: !!pkg.workspaces,
```

Comment:
I don't see where this field is used, is it a leftover from something that changed?

Reply:
oh yeah def a leftover, will clean it up 😊